apps/codeforafrica/next.config.js

```diff
@@ -12,9 +12,9 @@ const outputFileTracingRoot = PROJECT_ROOT
   ? path.resolve(__dirname, PROJECT_ROOT)
   : undefined;
 
-const ghostUrl =
-  process.env.GHOST_ADMIN_URL?.trim() || process.env.GHOST_URL?.trim();
-const ghostAdminUrl = new URL("/ghost", ghostUrl).toString();
+// const ghostUrl =
+//   process.env.GHOST_ADMIN_URL?.trim() || process.env.GHOST_URL?.trim();
+// const ghostAdminUrl = new URL("/ghost", ghostUrl).toString();
```

Comment:
No commented out code in repo. `git log -p` can show needed history.

Reply:
Yes, I commented this out when testing why CI was failing and forgot to change it back.